### ISSUE TITLE
[BUG] CRPS not defined in get_method function in verification interface 

### ISSUE THREAD

**Issue by pkars** [pySTEPS/pysteps#9]
Variable CRPS is not defined in get_method() function of pysteps.verification.interface module. Trying to call function with name='crps' raises NameError.

**Steps to reproduce:**

In [1]: import pysteps

In [2]: pysteps.verification.get_method('crps', 'probabilistic')

NameError                                 Traceback (most recent call last)
<ipython-input-2-6e2390ed7e23> in <module>()
----> 1 pysteps.verification.get_method('crps', 'probabilistic')

~/Nowcasting/pysteps/pysteps/verification/interface.py in get_method(name, type)
    142 
    143         if name in ["crps"]:
--> 144             return CRPS
    145         elif name in ["reldiag"]:
    146             return reldiag_init, reldiag_accum, reldiag_compute

NameError: name 'CRPS' is not defined

**Comment by dnerini** [comment]
Thank you for the bug report, @pkars. I have fixed it in commit 5e13ed98ce5b2dfbbc363c088191503b5ed18491,

**Comment by pkars** [comment]
Thank you @dnerini for this bugfix. However, commit https://github.com/pySTEPS/pysteps/commit/5e13ed98ce5b2dfbbc363c088191503b5ed18491 introduces a new fatal bug: `import pysteps` fails and raises `IndentationError: unexpected indent` at line 128 of pysteps.verification.interface.py .

Fix this by adding parenthesis around imported functions at lines 127-128 and 142-143. For example:
```
from .ensscores import (ensemble_skill, ensemble_spread, rankhist_accum, 
                        rankhist_compute)
```

**Comment by dnerini** [comment]
Gosh, I do need to get my 10 am coffee...

Thanks again, hopefully this is definitely solved now with commit 43be1660959d462299d98a74a4c45506393e3f06.
